SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -76,6 +77,8 @@
   @Min(value = 1, message = "Must be positive and non-zero")
   private int defaultBounceExpirationMinutes = 60;
 
+  private String defaultDatePattern = "MMM dd h:mm:ss a zzz";
```

Comment:
I'm not sure if they're going to be used for mail only or not - it's plausible that we would want to use them for something else. I don't know what that thing would be though.